content/en/docs/concepts/glossary.md

```diff
@@ -36,9 +36,8 @@ alerting and tooling for finding the root cause.
 
 ### **Attribute**
 
-A key-value pair. Used across telemetry signals - e.g. in [`Traces`](#trace) to
-attach data to a [`Span`](#span), or in [`Metrics`](#metric). See [attribute
-spec][attribute].
+See [`Metadata`](#metadata). Used across [`Signals`](#signal) and [`Resources`]
```

Comment:
Hmm, I think it might still be worth it to describe what an attribute looks like, and the fact that some vendors call them "tags" still. WDYT?

Reply:
I'm trying to avoid saying key-value pair everywhere otherwise, it doesn't seem to make sense to have a definition for or use the word metadata - thoughts? When the problems I see is that attributes is the standard here and neither is dimension, labels, or fields. At the end of the day, all of this is metadata.

Reply:
Could we describe it as a <something> that describes data and might take the form of a KV pair? Maybe with an example?

Reply:
OK, done!